docs/en/connector/sink/Jdbc.mdx

```diff
@@ -166,6 +166,7 @@ Sink plugin common parameters, please refer to [Sink Plugin](common-options.md)
 
 ```bash
 jdbc {
+    driver = "com.mysql.jdbc.Driver",
     saveMode = "update",
```

Comment:
We are using mysql-jdbc 8 by default. Can the driver name be changed to `com.mysql.cj.jdbc.Driver` for consistency?

Reply:
OK